android/modules/ui/src/java/ti/modules/titanium/ui/TabGroupProxy.java

```diff
@@ -367,7 +368,11 @@ public void windowCreated(TiBaseActivity activity, Bundle savedInstanceState)
 				((TiUITabLayoutTabGroup) view).setTabMode((Integer) getProperty(TiC.PROPERTY_TAB_MODE));
 			}
 		} else {
-			view = new TiUIBottomNavigationTabGroup(this, activity);
+			if (TiConvert.toBoolean(getProperty("newLayout"), false)) {
```

Comment:
Not necessary :) Let's be pragmatic here

Reply:
it's for testing. I only started it like 2 hours ago

Reply:
Alright! Just leaving early feedback here, hehe!

Reply:
:smile: no problem, always welcome!

![Screenshot_20241002-131331](https://github.com/user-attachments/assets/00ad039e-2b89-4b95-b79b-a590018f2fac)

have it working with the XML and the old TabGroup now. Still don't understand why I can go from M2 -> M3 but not back to M2 again when running Liveview. It looks like the XML is caching the layout somewhere. It's a minor issue I'll look at at the end again...features first (events, properties, colors)